### ISSUE TITLE
wxLua says running program in the console but nothing displays

### ISSUE THREAD

**Issue by kestermckinney** [pkulchenko/wxlua#74]
wxLua compiles on MacOS 10.14 just fine.  I open a file to run it and is says it is running and gives a process id.  However nothing shows.  If I copy the text after "Running program:", and run it in a terminal it works fine.  I'm not sure why wxLua can't run it.  Here is what it says and what works in a terminal.

/Applications/wxLua/wxLua.app/Contents/MacOS/wxLua -c /Users/paulmckinney/Documents/wxLua/install_files/share/wxlua/samples/gridtable.wx.lua 

Thanks for any help.
<img width="1482" alt="Screen Shot 2020-07-15 at 12 01 30 PM" src="https://user-images.githubusercontent.com/52587773/87568070-113e2900-c693-11ea-8820-db4520a7bfff.png">

**Comment by pkulchenko** [comment]
I'm not sure, as I don't support this executable. What do you see in the About screen of this application? If the command executes correctly from the terminal, this is all you need to get a wxwidgets application to run. Are you interested specifically in wxLua, because you need this executable for some project?

**Comment by kestermckinney** [comment]
It says wxLua 3.0.0.8 with wxWidgets 3.1.3.  I have built it with the dynamic libraries of wxWidgets.  I downloaded the Lua 5.1.5 library and binaries from source forge, so I built and linked it with the dynamo Lua 5.1.5 library.  I'm not sure if I'll use this app.  However, I think this issue may be symptomatic of another issue I'm experiencing.  I'll post that issue instead.  Maybe we can just close this one out.
